prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1868,7 +1880,12 @@ static inline void rdm_ep_poll_ibv_cq(struct rxr_ep *ep,
 				"Unhandled cq type\n");
 			assert(0 && "Unhandled cq type");
 		}
-	}
+
+		err = ibv_next_poll(efa_cq->ibv_cq_ex);
+	} while (true);
```

Comment:
`while(true)` is ugly. I understand the intention of using do while is to reuse the code that covert rdma-core return code to libfabric return code. Can we wrap that part in an inline function? which `efa_cq_readfrom` can also use.